proposals/003-installer-guidelines.md

```diff
@@ -0,0 +1,102 @@
+## Introduction
+
+The Haskell Foundation would like see a central Haskell download page in which the Haskell curious
+could easily download and run a installer to provide a standard Haskell environment that supports
+the principal build tools (`cabal-install` and `stack`) on the main platforms (Linux, macOS and
+Windows). All Haskell introductory material (text books, blog posts, etc.) should be able to point
+to the Haskell Foundation download page and work on the assumption that it will yield their
+widely-supported Haskell development environment.
+
+To this end this proposal covers two things:
+
+  * To publish a set of minimal requirements for a Haskell installer that we are prepared to
```

Comment:
I agree, the requirements should be *minimal* and I think we need to work on that.